pkg/controller/kieapp/webconsole/webconsole.go

```diff
@@ -0,0 +1,102 @@
+package webconsole
```

Comment:
Maybe just call it olm?

Reply:
probably not as all these customizations are meant for OCP Container platform and independent of the olm.

Reply:
Sure, I guess webconsole is fine